js/output/pjs/pjs-code-injector.js

```diff
@@ -80,8 +80,9 @@ class PJSCodeInjector {
             this.initializeProps();
         }
 
+        // TODO(kevinb) add options for the initial timeout and frame timeout
         this.loopProtector = new LoopProtector(
-            infiniteLoopCallback, 2000, 500, true);
+            infiniteLoopCallback, 1000, 500, true);
```

Comment:
The tests are looking stable after changing this to 1000.  The issues was that 2000ms is the same timeout used by mocha so sometimes the mocha timeout occurred first and sometimes it didn't.